core/vm/actor/builtin/types/miner/monies.hpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "vm/actor/builtin/states/miner/miner_actor_state.hpp"
+#include "common/outcome.hpp"
+#include "common/smoothing/alpha_beta_filter.hpp"
+#include "const.hpp"
+#include "primitives/types.hpp"
+#include "vm/actor/builtin/types/miner/policy.hpp"
+#include "vm/runtime/runtime.hpp"
+#include "vm/version/version.hpp"
+
+namespace fc::vm::actor::builtin::types::miner {
+  using common::smoothing::FilterEstimate;
+  using primitives::BigInt;
+  using primitives::ChainEpoch;
+  using primitives::StoragePower;
+  using primitives::TokenAmount;
+  using states::MinerActorState;
```

Comment:
```suggestion
  using states::MinerActorStatePtr;
```

Reply:
And use MinerActorStatePtr instead of Universal<MinerActorState> in all places

Reply:
fixed